reputation/reputation_test.go

```diff
@@ -0,0 +1,662 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+package reputation
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestCreateTable(t *testing.T) {
+	db, _ := startDB("./TestCreateTable.db")
+
+	createStmt := `
+	CREATE table foo (id interger not null primary key, name text);
+	`
+	createTable(createStmt, db)
+
+	os.Remove("./TestCreateTable.db")
+}
+
+func TestRowInsertAndQuery(t *testing.T) {
+	db, _ := startDB("./TestRowInsertAndQuery.db")
+
+	createStmt := `
+	CREATE table reputation (
```

Comment:
the sql statements are definitely part of the main application logic and not just part of the tests. if the sql statements are only part of the tests, then the tests are only validating that the application logic works against these specific sql statements, but don't validate that anything else works. i'd say move the sql statements out of the test file.